examples/nodejs/cache/doc-example-files/doc-examples-js-apis.ts

```diff
@@ -451,162 +481,192 @@ async function example_API_GetBatch(cacheClient: CacheClient, cacheName: string)
 async function example_API_ListFetch(cacheClient: CacheClient, cacheName: string) {
   await cacheClient.listConcatenateBack(cacheName, 'test-list', ['a', 'b', 'c']);
   const result = await cacheClient.listFetch(cacheName, 'test-list');
-  if (result instanceof CacheListFetch.Hit) {
-    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-    console.log(`List fetched successfully: ${result.valueList()}`);
-  } else if (result instanceof CacheListFetch.Miss) {
-    console.log(`List 'test-list' was not found in cache '${cacheName}'`);
-  } else if (result instanceof CacheListFetch.Error) {
-    throw new Error(
-      `An error occurred while attempting to fetch the list 'test-list' from cache '${cacheName}': ${result.errorCode()}: ${result.toString()}`
-    );
+  switch (result.type) {
+    case CacheListFetchResponse.Hit:
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      console.log(`List fetched successfully: ${result.valueList()}`);
```

Comment:
Is there a way to write this line so we can remove the eslint suppression from the docs?